week-4-practise/level-2/exercise-1.js

```diff
@@ -0,0 +1,38 @@
+/*
+Create a bottle object and give it the ability to be filled, 
+drank from and a function that checks whether it is empty.
+To measure this your bottle will need a volume along with fill, drink and empty keys.
+We have made a start on this for you here:  
+*/

```

Comment:
> Create an object that acts like a water bottle. It will need a volume key to store how full or empty the bottle is. It will be 100 when full and 0 when empty. Give your water bottle functions for filling it up, drinking some of it, and emptying it.
>
> We made a start on this for you here: